modules/460-log-shipper/monitoring/prometheus-rules/log-shipper-agent.yaml

```
@@ -46,8 +46,16 @@
       description: |
         A number of log-shipper-agents are not scheduled.
 
-        Consider checking state of the d8-log-shipper/log-shipper-agent DaemonSet.
-        `kubectl -n d8-log-shipper get daemonset,pod --selector=app=log-shipper-agent`
+        Consider checking state of the d8-log-shipper/log-shipper-agent DaemonSet:
+        `kubectl -n d8-log-shipper get daemonsets --selector=app=log-shipper`
+
+        Consider checking state of the d8-log-shipper/log-shipper-agent Pods:
+        `kubectl -n d8-log-shipper get pods --selector=app=log-shipper-agent`
```

```suggestion
        To check the state of the `d8-log-shipper/log-shipper-agent` DaemonSet:
        ```shell
        kubectl -n d8-log-shipper get daemonsets --selector=app=log-shipper
        ```

        To check the state of the d8-log-shipper/log-shipper-agent Pods:
        ```shell
        kubectl -n d8-log-shipper get pods --selector=app=log-shipper-agent
        ```